### PR TITLE
release v7.0.0-alpha.8

### PR DESCRIPTION
**Pull request by alexfauquette**
2. Clean the generated changelog, to match the format of [https://github.com/mui/mui-x/releases](https://github.com/mui/mui-x/releases).
3. Update the root `package.json`'s version
4. Update the versions of the other `package.json` files and of the dependencies with `yarn release:version` (`yarn release:version prerelease` for alpha / beta releases).
5. Open PR with changes and wait for review and green CI.
6. Merge PR once CI is green, and it has been approved.

### Release the packages

1. Checkout the last version of the working branch
2. `yarn && yarn release:build` (make sure you have the latest dependencies installed, and build the packages)
3. `yarn release:publish` (release the versions on npm, you need your 2FA device)
4. `yarn release:tag` (push the newly created tag)

### Publish the documentation

The documentation must be updated on the `docs-vX` branch (`docs-v4` for `v4.X` releases, `docs-v5` for `v5.X` releases, ...)

Push the working branch on the documentation release branch to deploy the documentation with the latest changes.

```bash
git push -f upstream next:docs-next
```

You can follow the deployment process [on the Netlify Dashboard](https://app.netlify.com/sites/material-ui-x/deploys?filter=docs-next)
Once deployed, it will be accessible at https://material-ui-x.netlify.app/ for the `docs-next` deployment.

### Publish GitHub release

After documentation is deployed, publish a new release on [GitHub releases page](https://github.com/mui/mui-x/releases)

### Announce

Follow the instructions in https://mui-org.notion.site/Releases-7490ef9581b4447ebdbf86b13164272d.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,81 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.0.0-alpha.8
+
+_Jan 11, 2024_
+
+We'd like to offer a big thanks to the 7 contributors who made this release possible. Here are some highlights ✨:
+
+- ⏰ Support date-fns v3 (#11659) @LukasTy
```
Comment (flaviendelangle) [inline CHANGELOG.md:12]: Same comment as for the path release on both highlights

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,81 @@
...
+
+### Tree View / `@mui/x-tree-view@7.0.0-alpha.8`
+
+- [tree view] Cleanup `onKeyDown` handler (#11481) @flaviendelangle
+- [tree view] Define the parameters used by each plugin to avoid listing them in each component (#11473) @flaviendelangle
+
+### Docs
+
+- [docs] Fix parsing of `x-date-pickers-pro` demo adapter imports (#11628) @LukasTy
+- [docs] Improve git diff format @oliviertassinari
```
Comment (flaviendelangle) [inline CHANGELOG.md:67]:
```suggestion
- [docs] Improve `git diff` format @oliviertassinari
```

File: CHANGELOG.md (hunk 1 above)
Comment (LukasTy) [reply CHANGELOG.md:12]: ⏰ Support date-fns v3 (#11462) @LukasTy
  Pickers support both v2 and v3 of date-fns. For v3 use `AdapterDateFnsV3`.
  ```js
  // with date-fns v2.x
  import { AdapterDateFns } from '@mui/x-date-pickers/AdapterDateFns';
  import de from 'date-fns/locale/de';
  ```
  ```js
  // with date-fns v3.x
  import { AdapterDateFns } from '@mui/x-date-pickers/AdapterDateFnsV3';
  import { de } from 'date-fns/locale/de';
  ```

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,81 @@
...
+  +import { enUS } from '@mui/x-data-grid-pro/locales';
+
+  -import { enUS } from '@mui/x-data-grid-premium';
+  +import { enUS } from '@mui/x-data-grid-premium/locales';
+  ```
+
+#### `@mui/x-data-grid@7.0.0-alpha.8`
+
+- [DataGrid] Stop exporting locales from the package root (#11614) @cherniavskii
+- [l10n] Add missing Danish (da-DK) locale export (#11640) @etlos
```
Comment (LukasTy) [inline CHANGELOG.md:35]:
```suggestion
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,81 @@
...
+
+
+#### `@mui/x-data-grid-premium@7.0.0-alpha.8` [![premium](https://mui.com/r/x-premium-svg)](https://mui.com/r/x-premium-svg-link 'Premium plan')
+
+Same changes as in `@mui/x-data-grid-pro@7.0.0-alpha.8`.
+
+### Date Pickers
+
+#### `@mui/x-date-pickers@7.0.0-alpha.8`
+
```
Comment (LukasTy) [inline CHANGELOG.md:49]:
```suggestion

- [l10n] Add missing Danish (da-DK) locale export (#11640) @etlos
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -3,6 +3,81 @@
...
+- [pickers] Add date-fns@3.x adapter (#11462) @LukasTy
```
Comment (LukasTy) [inline CHANGELOG.md:50]:
```suggestion
- [pickers] Add `date-fns@3.x` adapter (#11462) @LukasTy
```

File: packages/grid/x-data-grid-premium/package.json [hunk 6]
```diff
@@ -46,9 +46,9 @@
     "@babel/runtime": "^7.23.7",
     "@mui/system": "^5.15.2",
     "@mui/utils": "^5.15.2",
-    "@mui/x-data-grid": "7.0.0-alpha.7",
-    "@mui/x-data-grid-pro": "7.0.0-alpha.7",
-    "@mui/x-license-pro": "7.0.0-alpha.1",
+    "@mui/x-data-grid": "7.0.0-alpha.8",
+    "@mui/x-data-grid-pro": "7.0.0-alpha.8",
+    "@mui/x-license-pro": "7.0.0-alpha.2",
```
Comment (LukasTy) [inline packages/grid/x-data-grid-premium/package.json:51]: We usually sync versions of all the packages when releasing.
```suggestion
    "@mui/x-license-pro": "7.0.0-alpha.8",
```

File: packages/grid/x-data-grid-pro/package.json [hunk 7]
```diff
@@ -46,8 +46,8 @@
     "@babel/runtime": "^7.23.7",
     "@mui/system": "^5.15.2",
     "@mui/utils": "^5.15.2",
-    "@mui/x-data-grid": "7.0.0-alpha.7",
-    "@mui/x-license-pro": "7.0.0-alpha.1",
+    "@mui/x-data-grid": "7.0.0-alpha.8",
+    "@mui/x-license-pro": "7.0.0-alpha.2",
```
Comment (LukasTy) [inline packages/grid/x-data-grid-pro/package.json:50]:
```suggestion
    "@mui/x-license-pro": "7.0.0-alpha.8",
```

File: packages/x-date-pickers-pro/package.json [hunk 8]
```diff
@@ -46,8 +46,8 @@
     "@mui/base": "^5.0.0-beta.29",
     "@mui/system": "^5.15.2",
     "@mui/utils": "^5.15.2",
-    "@mui/x-date-pickers": "7.0.0-alpha.7",
-    "@mui/x-license-pro": "7.0.0-alpha.1",
+    "@mui/x-date-pickers": "7.0.0-alpha.8",
+    "@mui/x-license-pro": "7.0.0-alpha.2",
```
Comment (LukasTy) [inline packages/x-date-pickers-pro/package.json:50]:
```suggestion
    "@mui/x-license-pro": "7.0.0-alpha.8",
```

**Review by LukasTy — APPROVED** [review]

**Comment by oliviertassinari** [comment]
Something strange happened on this one. https://next.mui.com/x/react-tree-view/ gave me a 404 (I was following a link on Twitter). Looking closer

It's because we pushed v6.0.0 to the branch. Proof https://github.com/mui/mui-x/activity?ref=docs-next:



I fixed it.

**Comment by LukasTy** [comment]
Not sure what was the problem, but `docs-v6` was also 2 months old and on `6.18.0`. 🤷 🙈 
I've pushed the latest master there and now the doc is up to date.

**Pull request merged**